configstack/running_module.go

```diff
@@ -249,7 +249,7 @@ func (module *runningModule) moduleFinished(moduleErr error) {
 	if moduleErr == nil {
 		module.Module.TerragruntOptions.Logger.Printf("Module %s has finished successfully!", module.Module.Path)
 	} else {
-		module.Module.TerragruntOptions.Logger.Printf("Module %s has finished with an error: %v", module.Module.Path, moduleErr)
+		module.Module.TerragruntOptions.Logger.Printf("Module %s has finished with status: %v", module.Module.Path, moduleErr)
```

Comment:
This doesn't seem like the right approach, as this will affect all commands/args, and not just `plan-all --detailed-exit-code`.